src/Constants.ts

```diff
@@ -566,6 +566,10 @@ export const TriggerHeaderTags = {
     RESOURCE_NAME: 'x-thundra-resource-name',
 };
 
+export const IntegrationSpecialHeaders = [
+    'x-elastic-client-meta',
```

Comment:
What is this header for and when it is used?

Reply:
According to here: https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/changelog-client.html#_added_x_elastic_client_meta_header_1373
> Adds the x-elastic-client-meta HTTP header which is used by Elastic Cloud and can be disabled with the enableMetaHeader parameter set to false.

So seems that it has been added since version `7.11`. So what will happen for versions until `7.11` (`7.0.x` ... `7.10.x`)?